Dockerfile

```diff
@@ -52,6 +51,11 @@ ENV LANG=C.UTF-8
 
 WORKDIR /home/plasma/elixir-omg/
 
+RUN wget https://github.com/ethereum/solidity/releases/download/v0.4.23/solc-static-linux \
```

Comment:
Jenkins builds use 0.4.24:
https://github.com/omisego/elixir-omg/blob/develop/Jenkinsfile
https://hub.docker.com/r/omisegoimages/blockchain-base/~/dockerfile/

alternatively we could consistently do 0.4.25, which seems to be the recent 0.4

Reply:
Version bumped to 0.4.25